ROBOT:
  hardware_interface:
     loop_hz: 0.25 # hz
     joints:
       - YOUR_ROBOT_JOINT
